rust/share/agama.libsonnet

```diff
@@ -0,0 +1,26 @@
+// function go throught lshw output and enlist only given class.
+// Basically it is same as calling `lshw -class <class>`.
+selectClass(lshw, class)::
```

Comment:
`selectClass` looks good enough. But perhaps to be consistent with my suggestion below we could use `findClass` or something similar.jsdo

Reply:
Please, document the arguments.